packages/core/src/Stores/Helpers/client-notifications.js

```diff
@@ -18,6 +18,12 @@ import { WS } from 'Services';
 /* eslint-disable react/jsx-no-target-blank */
 export const clientNotifications = (ui = {}, client = {}) => {
     const notifications = {
+        dp2p: {
+            key: 'dp2p',
+            header: localize('Payment problems?'),
+            message: localize('There’s an app for that'),
+            type: 'dp2p',
```

Comment:
maybe we can make it type `news` etc? so in the future if we have more pop ups like this that are just announcing our new products, we can use the same type and it'll work the same way

Reply:
Understood, will refactor the P2P banner component into a general notification banner for `news` so that any new products can utilise it as well.